one-ui/src/test/java/com/marklogic/hub/oneui/TestHelper.java

```diff
@@ -102,4 +122,20 @@ public void addStagingDoc(String uri, DocumentMetadataHandle meta, String resour
     public File getResourceFile(String resourceName) {
         return new File(Objects.requireNonNull(TestHelper.class.getClassLoader().getResource(resourceName)).getFile());
     }
+    private void createUser(String username, String password, String role) {
+        client = new ManageClient();
+        client.setManageConfig(new ManageConfig(mlHost, 8002, adminUserName, adminPassword));
+        adminAPI = new API(client);
+
+        user = new User(adminAPI, username);
+        user.setUserName(username);
+        user.setPassword(password);
+        user.setRole(Stream.of(role).collect(Collectors.toList()));
+        user.save();
+    }
+
+    @AfterAll
```

Comment:
Is this actually called? I believe this method must be static - see https://howtodoinjava.com/junit5/after-all-annotation-example/ . 

I also don't think there's any need to delete the user, it can be considered part of the test setup. Just make sure it has a name that makes it clear it's for running tests.